.github/workflows/nightly_release.yml

```diff
@@ -47,15 +50,23 @@ jobs:
           # Install node dependencies
           node ./scripts/gtime.js monorepo_install npm ci
 
+      - name: Update identity in git config
+        run: |-
+          git config --global user.name ${{ secrets.GIT_CONFIG_USERNAME }}
+          git config --global user.email ${{ secrets.GIT_CONFIG_EMAIL }}
+
       - name: Create Release Branch
-        run: git switch release-nightly-${{ env.monorepo_version_major }}.x || git switch -c release-nightly-${{ env.monorepo_version_major }}.x
+        run: |-
+          git fetch
```

Comment:
Github action runs in total isolation and is unaware of existence of other branches except the one that triggered the workflow. So we git fetch all other branches.